pyproject.toml

```diff
@@ -107,15 +106,7 @@ rm -rf
     ./**/.ruff_cache
     ./**/.tox
 """
-mdformat = """
-mdformat
-    CODE_OF_CONDUCT.md
-    CONTRIBUTING.md
-    README.md
-    SECURITY.md
-    tests/README.md
-    codegen/README.md
-"""
+mdformat = "mdformat ."
```

Comment:
I usually use the lowest supported python version when developing, which can help catch bugs. And since `exclude` requires `python>=3.13`, the markdown files need to be explicitly passed for `mdformat` to work on `python<3.13`. I agree that it's not DRY and ugly, but in this case it's a necessary workaround. 

You could consider submitting a PR to `mdformat` that fixes it; I can't imagine it being all that difficult to implement.

Reply:
Another option here would be to remove the `mdformat` dependency (and its plugins) from the dev uv deps, keeping as a separate group:
```toml
[dependency-groups]
mdformat = ["mdformat", "plugins"]
```

Then, change the `poe` command to use a self-contained env made on purpose for the MD check:
```bash
uv run --isolated --group=mdformat --python=3.13 mdformat .
```

Reply:
Hmm, I think I prefer a list of explicit markdown files, as this will slow down stuff like pre-commit, which is bad for DX.

Reply:
Do you want this reverted before merging?

Reply:
Yea I think that would be best. And then maybe add a comment that says this is a workaround for the missing mdformat `exclude`with `python<3.13`.